docs/operations/README.md

```diff
@@ -45,6 +45,8 @@ When deploying to your personal sandbox, you should make sure all of the USWDS a
 
 For ease of use, you can run the `deploy.sh <sandbox name>` script in the `/src` directory to build the assets and deploy to your sandbox. Similarly, you could run `build.sh <sandbox name>` script to just compile and collect the assets without deploying.
 
+You may also manually deploy to a sandbox using our [manual deploy workflow](https://github.com/cisagov/manage.get.gov/actions/workflows/deploy-manual.yaml) on GitHub Actions. Select Run workflow and enter the branch you want to deploy to your sandbox of choice.
```

Comment:
This link will only work once we merge since we're renaming our workflow from deploy-branch-to-sandbox to deploy-manual